conan.lock

```diff
@@ -17,7 +17,7 @@
         "libarchive/3.8.1#ffee18995c706e02bf96e7a2f7042e0d%1764175360.142",
         "jemalloc/5.3.0#e951da9cf599e956cebc117880d2d9f8%1729241615.244",
         "grpc/1.50.1#02291451d1e17200293a409410d1c4e1%1756234248.958",
-        "ed25519/2015.03#17c1f1910e769f368025267a53c23c13%1764259445.491",
+        "ed25519/2015.03#ae761bdc52730a843f0809bdf6c1b1f6%1764270189.893",
```

Comment:
This is due to https://github.com/XRPLF/conan-center-index/pull/25, nothing to worry about